docs/modules/ROOT/pages/connect-outside-kubernetes.adoc

```diff
@@ -156,6 +157,9 @@ const clientConfig = {
     clusterMembers: [
       '35.230.92.217'
     ]
+  },
+  properties: {
```

Comment:
@hasancelik 
I added the corresponding property `hazelcast.discovery.public.ip.enabled` in nodejs client.